### PR TITLE
BUGFIX: PDOBackend doesn't respect configuration

### PR DESCRIPTION
**Pull request by Sebobo**
**What I did**

* Use the table names from the configuration in all places.

### CODE REVIEW

**Comment by Sebobo** [comment]
Relates also to #1764

File: Neos.Cache/Classes/Backend/PdoBackend.php [hunk 1]
```diff
@@ -518,6 +518,12 @@ public function valid(): bool
      */
     public function rewind()
     {
+        try {
+            $this->connect();
+        } catch (Exception $e) {
+            return;
+        }
```
Comment (bwaidelich) [inline Neos.Cache/Classes/Backend/PdoBackend.php]: Is that related?

Comment (bwaidelich) [reply Neos.Cache/Classes/Backend/PdoBackend.php]: ah, you mention it in the PR description..
I always struggle a bit when I see a `catch` block that is ignored.

Comment (Sebobo) [reply Neos.Cache/Classes/Backend/PdoBackend.php]: Should maybe have been a separate PR.

Comment (Sebobo) [reply Neos.Cache/Classes/Backend/PdoBackend.php]: Alternative would be to override getIterator and call connect there.
